### DISCUSSION TITLE
eza

### DISCUSSION THREAD

**Discussion by j-xella** [catppuccin/catppuccin#2220] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

The Github home page is [here](https://github.com/eza-community/eza)

In the `man` folder, there are pages with explanations regarding the coloring.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by maddawik** [comment]
You can achieve this using the `LS_COLORS` or `EZA_COLORS`  env vars (see the `man` page, would definitely suggest `LS_COLORS` so that other tools that respect it would also benefit, `dust` is another that comes to mind). To generate the colors for catppuccin you can use something like [vivid](https://github.com/sharkdp/vivid)

After installing with your package manager:

```sh
# bash, zsh
export LS_COLORS="$(vivid generate catppuccin-mocha)"

# fish
set -Ux LS_COLORS $(vivid generate catppuccin-mocha)
```

**Comment by nullishamy** [comment]
Closing as complete as based on the guidance above. Feel free to reopen if there's something missing still!
